tests/test_secondary.py

```diff
@@ -0,0 +1,721 @@
+"""
+<Program Name>
```

Comment:
Adding, but it would be good to decide in a central location if this is something we should be doing.